Scripts/Hooks/scratchpad-hook.lua

```diff
@@ -260,9 +260,11 @@ local function loadScratchpad()
     local function coordsType()
         local ac = DCS.getPlayerUnitType()
         if ac == "FA-18C_hornet" then
-            return "DMS", true
-        elseif ac == "A-10C_2" then
+            return "DMS/DDM", true
```

Comment:
I am pretty sure that the precisce bombing of the F18 uses DMS. You enter degrees, minutes, seconds and then decimal seconds.

Example:
```
N 13°35'12.52", E 144°55'54.11"
N 13°35.208', E 144°55.901'
55 P BR 76205 02979
168m, 551ft
```

Is entered as follows (taing lat as the example):
![image](https://user-images.githubusercontent.com/409021/176188919-2ac026da-e896-4bec-80dd-078ed435d3a7.png)
![image](https://user-images.githubusercontent.com/409021/176188995-5fc04d05-3bde-47a7-a2ed-281ac75bd446.png)

So I don't think that there is a change necessary for the F18, unless I am missing something?